packages/web3torrent/src/constants.ts

```diff
@@ -17,6 +17,13 @@ export const BUFFER_REFILL_RATE = utils.bigNumberify(WEI_PER_BYTE.mul(BLOCK_LENG
 
 export const INITIAL_SEEDER_BALANCE = utils.bigNumberify(0); // needs to be zero so that depositing works correctly (unidirectional payment channel)
 
+const VERSION_STR = '0.7.2'.replace(/\d*./g, v => `0${Number(v) % 100}`.slice(-2)).slice(0, 4);
```

Comment:
This is.... eye-catching. Could you add a comment explaining what is going on here?

Reply:
sure! BTW, this is a rewrite from [here](https://github.com/webtorrent/webtorrent/blob/ed2809159585d2611dff24a48f25748baf78e8fb/index.js#L52)

Reply:
https://github.com/statechannels/monorepo/pull/1449/commits/cc79225268d6337a8d5eae8e7764788d3d583483